src/ifcparse/IfcParse.cpp

```diff
@@ -70,8 +70,8 @@ void init_locale() {
 // Opens the file, gets the filesize and reads a chunk in memory
 //
 IfcSpfStream::IfcSpfStream(const std::string& fn)
-    : stream(0)
```

Comment:
BTW the convention will be to use spaces instead of tabs, hence I've already started to use spaces, https://github.com/IfcOpenShell/IfcOpenShell/issues/21